lightcone/animate.py

```diff
@@ -1,6 +1,8 @@
 from lightcone.plot_3d import Plot3d
+from tqdm import tqdm
```

Comment:
you changed the file name but somehow this change was not recognized in the test folder to be renamed. Make sure you re-name the file in the test import statement such that the continuous integration is not failing there.